packages/firestore/rollup.config.js

```diff
@@ -45,8 +45,11 @@ import {
 // The in-memory builds are roughly 130 KB smaller, but throw an exception
 // for calls to `enablePersistence()` or `clearPersistence()`.
 //
-// All browser builds rely on Terser's property name mangling to reduce code 
+// All browser builds rely on Terser's property name mangling to reduce code
 // size.
+//
+// See https://g3doc.corp.google.com/firebase/jscore/g3doc/contributing/builds.md
```

Comment:
FYI, you can shorten this to `http://g3doc/firebase/jscore/g3doc/contributing/builds.md` if you wanted.

Reply:
Done